docs/writing-docs/mdx.md

```diff
@@ -218,6 +219,36 @@ paths={[
 
 ![MDX docs only story](./mdx-documentation-only.png)
 
+### Linking to other stories and pages
+
+Another way to improve documentation is by linking to other stories and pages. Suppose you already have a component story with the following unique identifier, `some--id`, and you want to link it to your documentation page. In that case, you can use the `path` query string to redirect to the documentation entry related to the story:
+
+```md
+[Go to specific documentation page](?path=/docs/some--id)
```

Comment:
Hmm I don't know, but I can imagine linking between docs pages is a common pattern. What you're suggesting sounds like a new `<Link to={ButtonStories} />` block, which seems valuable to me.
Something to explore in upcoming releases for sure.

Reply:
OK, I guess we leave this for now, but I wonder if we should adding documentation that we probably plan to deprecate soon 🤷

Reply:
This was left in on a temporary basis until there's an API to address this to avoid the issues related to linking that continue to surface in the repo addressed. Once we have a stable API and a path moving forward, this section will be rewritten.